### DISCUSSION TITLE
Signing for Apple

### DISCUSSION THREAD

**Discussion by fooziex** [openaudible/openaudible#754] [Ideas]
Howdy,

I have an Apple developer account. Is there a reason I can't (or shouldn't) sign & notarize this project for you?

Thanks,
Tim Hofsommer

**Comment by openaudible** [comment]
That's a good idea. We would be interested.  Can you drop a note to openaudible [at] email.tg 
Also, if anyone reading this can help with a generic code cert, we'd love to talk as well.

**Comment by fooziex** [comment]
OK, email sent.

**Comment by openaudible** [comment]
Hey we got signed and notarized.. thank you for the offer!

**Reply by fooziex** [reply]
Awesome! Glad you got this worked out!
